### PR TITLE
add new tuya sensor

### PR DESCRIPTION
**Pull request by 13717033460**
1.Add new tuya millimeter wave human presence sensor, ZG-205Z、ZG-205ZL. 
2.The ZG-204ZL and ZG-102ZL add adjustable brightness acquisition time。

### CODE REVIEW

File: src/devices/lixee.js [hunk 1]
```diff
@@ -87,31 +87,37 @@ const fzLocal = {
             const result = {};
 
             const elements = [
-                /* 0x0305 */ 'totalReactivePower',
+                /* 0x0304 */ 'totalActivePower',
```
Comment (Koenkk) [inline src/devices/lixee.js:90]: Please revert the changes to this file.

**Comment by gijsh070** [comment]
Hi,

Is there support coming for the TS0601 _TZE200_2aaelwxk, I tried with the 225 converter but functions do not work. I does recoqnize the device after importing the .js file?

https://nl.aliexpress.com/item/1005006593707347.html?srcSns=sns_Copy&spreadType=socialShare&bizType=ProductDetail&social_params=60554233187&aff_fcid=202c6b5656de4acfbf640bd0fff1f171-1711396744440-00457-_EuhLli5&tt=MG&aff_fsk=_EuhLli5&aff_platform=default&sk=_EuhLli5&aff_trace_key=202c6b5656de4acfbf640bd0fff1f171-1711396744440-00457-_EuhLli5&shareId=60554233187&businessType=ProductDetail&platform=AE&terminal_id=26a5dddd4c32449da3685bf9a501e738&afSmartRedirect=y


Thanks in advance.
